news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md

```diff
@@ -0,0 +1,127 @@
+- [CODE:0](https://osu.ppy.sh/beatmapsets/1804396#mania/3760851) (RC3) @ [19:05](https://www.twitch.tv/videos/2233674722?t=00h19m05s) — Another breakpoint, this time from Singapore with all team members getting a score v1 SS.
+- [Hallucinate (Cut Ver.)](https://osu.ppy.sh/beatmapsets/2234584#mania/4746373) (RC2) @ [37:22](https://www.twitch.tv/videos/2233674722?t=00h37m22s) — Yet another breakpoint, this time from Poland with a miss from Singapore at the very end, letting them barely get the point.
+- [Senkou no Flare (Cut Ver.)](https://osu.ppy.sh/beatmapsets/2234576#mania/4746352) (RC5) @ [42:28](https://www.twitch.tv/videos/2233674722?t=00h42m28s) — Poland continued their momentum from the last pick and tied the match up 3–3 with each team member putting up insane scores while viewers were blown away.
+
+Unfortunately for Poland, Singapore managed to clutch up and take the match 5–3. While Singapore took the match this time around, the Polish will not be forgotten with their incredible performance and beating everyone’s expectations. We wish Poland the very best going into the losers' bracket of the tournament.
+
+## Score highlights
+
+Earlier rounds are known to be calmer as higher seeded teams get matched against their lower seeded peers. Even so, attention should be brought to a couple of note-worthy scores and players to keep track of:
+
+- We all know ::{ flag=CL }:: [myucchi](https://osu.ppy.sh/users/10072733) and their feats. While their performance across the entire mappool [vs. New Zealand](https://osu.ppy.sh/community/matches/115223051) was phenomenal, they crushed it in [Mono. - Tricolor Prizm☆☆☆ミ](https://osu.ppy.sh/beatmapsets/2024637#mania/4217619) with a score of 997,781.
+- Taking everyone by surprise, ::{ flag=VN }:: [Joki_](https://osu.ppy.sh/users/18043326) has let everybody know they're a force to be reckoned with. Their rice performance was second to none, as seen with their play on [SHRINE 418 - Little Little Princess](https://osu.ppy.sh/beatmapsets/2234579#mania/4746363), putting up a score of 997,904 [during their match vs. Ukraine](https://osu.ppy.sh/community/matches/115210872).
+- ::{ flag=JP }:: [CrewK](https://osu.ppy.sh/users/11488604) demonstrated yet again who's king, with an extraterrestrial performance on [Shpongle - Outer Shpongolia](https://osu.ppy.sh/beatmapsets/2234598#mania/4746405) during the Japan vs. France match, scoring an astonishing 998,578 points.
+
+Keep a look-out for these people! Let's see how future rounds fare them, and whether they can keep up the pace.
+
+---
+
+Now is the perfect time to start following the MWC4K, choose your favourites and cheer them on!
+
+Regardless of whom you'll be cheering on, we hope to see you this weekend for the Round of 16 on [Twitch](https://www.twitch.tv/osulive), and if you are interested in the nitty-gritty of this MWC4K edition, you can find the full details over on the [wiki page](/wiki/Tournaments/MWC/2024_4K)!
```

Comment:
i believe this is fairly important to always mention. linking the spreadsheet works too for this purpose if i'm not mistaken

```suggestion
Regardless of whom you'll be cheering on, we hope to see you this weekend for the Round of 16 on [Twitch](https://www.twitch.tv/osulive), and if you are interested in the nitty-gritty of this MWC4K edition (including match schedules), you can find the full details over on the [wiki page](/wiki/Tournaments/MWC/2024_4K)!
```

Reply:
everything's on the wiki so should be fine